PowerPointLabs/PowerPointLabs/ActionFramework/Image/Group/AnimationLabImageHandler.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Drawing;
+using PowerPointLabs.ActionFramework.Common.Attribute;
+using PowerPointLabs.ActionFramework.Common.Interface;
+
+namespace PowerPointLabs.ActionFramework.Image.Group
+{
+    [ExportImageRibbonId("AutoMotionGroup")]
```

Comment:
the id looks confusing, sometimes upper case sometimes lower case, any standard?